### PR TITLE
docs: clarify locust invocation norms

### PR DESCRIPTION
**Pull request by pwnage101**
this is an attempt to addresses #500

---

Push for the following norm: all module imports either reference
installed modules, or modules beneath the "project root" (i.e. the CWD).
To that end, remove any examples where the locustfile is not beneath the
CWD.

Also, provide some example project directory structures.

### CODE REVIEW

**Comment by pwnage101** [comment]
FYI, this new commit brings some typo fixes and other updates:

* clarify workaround with code snippet
* fix typos
* use double backticks correctly
* consistently refer to "locust files" using two words

**Comment by cgoldberg** [comment]
👍

**Comment by pwnage101** [comment]
@justiniso can you take a look?

File: docs/writing-a-locustfile.rst [hunk 1]
```diff
@@ -364,3 +364,59 @@ Example::
...
+  * ``locustfile_web_app.py``
+
+  * ``locustfile_api.py``
+
+  * ``locustfile_ecommerce.py``
+
+The locustfiles may import common libraries using, e.g. ``import commonlib_auth``.  This approach does not
+cleanly separate common libraries from locust files, however.
+
+Subdirectories can be a cleaner approach (see example below), but locust cannot currently cope with importing
```
Comment (justiniso) [inline docs/writing-a-locustfile.rst]: > but locust cannot currently cope with importing modules which live outside the locustfiles directory

I would rephrase this as:

> but locust will only import modules relative to the directory in which the running locustfile is placed. If you wish to import from your the project root (i.e. the path of the location where you are running the locust command), make sure to .....

**Comment by pwnage101** [comment]
@justiniso see updated text

**Comment by pwnage101** [comment]
Thanks! This now has two member approvals, please merge.

**Comment by pwnage101** [comment]
bahhh i just realized i hadn't squashed :(  so sorry...
